exercises/en/test_02_11.py

```diff
@@ -0,0 +1,14 @@
+def test():
+    # Here we can either check objects created in the solution code, or the
+    # string value of the solution, available as __solution__. A helper for
+    # printing formatted messages is available as __msg__. See the testTemplate
+    # in the meta.json for details.
+
+    # If an assertion fails, the message will be displayed
+
+    assert "data/pokemon.csv" in __solution__ , "Are you specifying the correct data path?"
+    assert "index_col=0" in __solution__ , "Are you making sure to add index_col=0 ?"
+    assert "sp_attack':'special_a" , "are you changing 'sp_attack' to 'special_a' using ':' "
+    assert "sp_defense':'special_a" , "are you changing 'sp_defense' to 'special_d' using ':' "
+    __msg__.good("Nice work, well done!")
+
```

Comment:
Can you check that they are using rename() and curly brackets? Are you doing any tests that are actually checking the column names in the object? 
>`assert "sp_attack':'special_a" , "are you changing 'sp_attack' to 'special_a' using ':' "`

What is this referencing? Is this checking the objects column names?